src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,9 @@
-// Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+import { TodoInfo } from '../TodoInfo';
+
+export const TodoList = ({ todos }) => (
+  <section className="TodoList">
+    {todos.map(todo => (
```

Comment:
Let's name it todoTask, so it will be very obvious what are we iterating through

Reply:
todoTask sound weird, don't you think? todo is a task itself, todoTask for me is the same as todoTodo or taskTask :))